app/src/main/java/com/android/voyageur/ui/authentication/SignIn.kt

```diff
@@ -40,23 +39,61 @@ import com.google.firebase.auth.auth
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.tasks.await
 
+@Composable
+private fun rememberFirebaseAuthLauncher(
+    onAuthComplete: (AuthResult) -> Unit,
+    onAuthError: (ApiException) -> Unit
+): ManagedActivityResultLauncher<Intent, ActivityResult> {
+  val scope = rememberCoroutineScope()
+
+  return rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) {
```

Comment:
Also the line coverage is a bit low as this part is not covered.

Reply:
I agree with you on the coverage, however it is really difficult to test these structures with Mockito. Could we leave this be? Since this is a relatively short piece of code and it would not really impact the global coverage